modules/apps/site/site-insurance-site-initializer/remote-web-components/raylife-d2c-web/src/utils/businessFields.js

```diff
@@ -0,0 +1,21 @@
+export const BCC_ALLOWED = {
+    PERCENT_SALES: ["750", "1349"],
```

Comment:
For now it's okay, but the same from previous: this should be dynamic in the future, isn't @joseabelenda ?